api/src/main/java/org/openmrs/module/insuranceclaims/InsuranceClaimsConfig.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.insuranceclaims;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * Contains module's config.
+ */
+@Component("insuranceclaims.InsuranceClaimsConfig")
+public class InsuranceClaimsConfig {
+
+	public static final String MODULE_PRIVILEGE = "Insurance Claims Privilege";
```

Comment:
I understand that we will add logic later ?

Maybe try to create a separate class for constant fields ?

Reply:
The module privileges are managed via the config.xml file. I've added missing privileges to that file.

I think we should put all constant fields related to privileges in that InsuranceClaimsConfig file and for the other constants create a separate class. I think this approach will improve the code readability and it could be some OpenMRS convention (because of this file was generated via the openMRS SDK).

Reply:
What do you think about that?